master/internal/webhooks/postgres_webhook.go

```diff
@@ -272,6 +280,121 @@ func (l *WebhookManager) deleteWebhook(ctx context.Context, id WebhookID) error
 	return nil
 }
 
+func handleCustomTriggerData(ctx context.Context, data CustomTriggerData, experimentID int, trialID *int) error {
+	var m struct {
+		bun.BaseModel `bun:"table:experiments"`
+		model.Experiment
+		ConfigBytes []byte `bun:"config"`
+	}
+	err := db.Bun().NewSelect().Model(&m).ExcludeColumn("username").Where("id = ?", experimentID).Scan(ctx)
+	if err != nil {
+		return fmt.Errorf("error getting experiment from id %d: %w", experimentID, err)
+	}
+
+	activeConfig, err := expconf.ParseAnyExperimentConfigYAML(m.ConfigBytes)
+	if err != nil {
+		return fmt.Errorf("error parsing experiment config : %w", err)
+	}
+	integrationsConfig := activeConfig.Integrations()
+	if integrationsConfig == nil {
+		return nil
+	}
+	webhookConfig := integrationsConfig.Webhooks
+	if webhookConfig == nil || webhookConfig.Exclude {
+		return nil
+	}
+
+	workspaceID, err := experiment.GetWorkspaceFromExperiment(ctx, &m.Experiment)
+	if err != nil {
+		return fmt.Errorf("error getting workspace from experiment : %w", err)
+	}
+
+	var es []Event
+	if webhookConfig.WebhookID != nil {
```

Comment:
Can we make the `WebhookID` plural?, so `WebhooksConfigV0.WebhookID` -> `WebhooksConfigV0.WebhookIDs` (and same with `WebhookName`)?

Reply:
These were defined at experiment configs, which were already merged. If we don't feel too strongly about it, maybe we can keep it like this?

Reply:
Makes sense, works for me!